src/file_worker.py

```diff
@@ -429,11 +414,12 @@ def get_dataset_file_infos(self, dataset_uuid, bearer_token=None):
         donors_dict_list = []
         for sample_uuid in entity_prov_info['dataset_samples'].keys():
             sample_dict = {}
-            specimen_type = entity_prov_info['dataset_samples'][sample_uuid]['specimen_type']
-            if specimen_type != 'organ':
+            sample_category = entity_prov_info['dataset_samples'][sample_uuid]['sample_category']
+            if sample_category != 'organ':
                 sample_dict['uuid'] = sample_uuid
-                sample_dict['code'] = specimen_type
-                sample_dict['type'] = self.tissue_type_dict[specimen_type]['description']
+                sample_dict['code'] = sample_category
+                # Repeat the 'code' value until a replacement for tissue_sample_type.yaml is available.
+                sample_dict['type'] = sample_category
```

Comment:
Do we want to capitalize the type? When we used the `tissue_sample_types.yaml`, the `description` field in the yaml is capitalized. The search-api is also capitalizing the value to be one of the "Section/Block/Suspension". Up to you.

Reply:
Sure, @yuanzhou I'll capitalize it if that is what search-api is doing.  I'll update momentarily, thanks!